example/test.js

```diff
@@ -1,13 +1,9 @@
-import { assert, config as chaiConfig } from "chai";
-chaiConfig.truncateThreshold = 0;
+import { assert, config, compile, toIntWith, solution } from "./lc-test.js";
 
-import * as LC from "@codewars/lambda-calculus";
-import { solution } from "./files.js"; // /workspace/files.js
-
-LC.config.purity = "Let";
-LC.config.numEncoding = "Church";
-const toInt = LC.toIntWith(LC.config);
-const { counter } = LC.compile(solution());
+config.purity = "Let";
+config.numEncoding = "Church";
+const toInt = toIntWith(config);
+const { counter } = compile(solution());
```

Comment:
Maybe `getSolution()` is better?

Reply:
I think `getSolution()` or `readSolution()` is better. It makes it more natural if an author needs to work with a solution text more than just compiling, using `const solution = readSolution()`